cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -146,6 +146,47 @@ def get_initial_simplex_and_names(
         """Return an initial Nelder-Mead simplex and the names for each parameter."""
 
 
+def _try_defaults_from_unitary(gate: 'cirq.Gate') -> Optional[Dict[str, 'cirq.TParamVal']]:
```

Comment:
(Optional since it's a private method) It might be nice to have a short docstring here.